Core/GDCore/IDE/CaptureOptions.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+#include <memory>
+#include <vector>
+
+#include "GDCore/String.h"
+
+namespace gd {
+
+class GD_CORE_API Screenshot {
+ public:
+  Screenshot();
+  virtual ~Screenshot() {};
+
+  void SetDelayTimeInSeconds(int delayTimeInMs_) {
+    delayTimeInMs = delayTimeInMs_;
+  }
+  const long GetDelayTimeInSeconds() const { return delayTimeInMs; }
```

Comment:
- No need for const in the return type for primitives
- You return a long but the type is a int. Return an int.